static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -708,26 +766,20 @@ require.def(
                 throw "ApiError: " + errorMessage;
             },
 
-            _setDisplayFullScreenForVideo: function() {
-                if (this._type === MediaPlayer.TYPE.VIDEO) {
-                    var dimensions = RuntimeContext.getDevice().getScreenSize();
-                    this.logger.info('Calling this.playerPlugin.SetDisplayArea');
-                    var result = this._playerPlugin.SetDisplayArea(0, 0, dimensions.width, dimensions.height);
-                    this.logger.info('"SetDisplayArea" responded with ' + result);
-                }
-            },
-
             _isSuccessCode: function(code) {
```

Comment:
This is only used in 1 place (`_tryPauseWithStateTransition`). If it doesn't need to be used elsewhere perhaps we can fold it into `_tryPauseWithStateTransition`.